src/js/components/Navigation/HeaderGettingStartedBar.jsx

```diff
@@ -114,10 +114,15 @@ export default class HeaderGettingStartedBar extends Component {
                                     show={this.state.showBallotIntroFollowIssues}
                                     onHide={()=>{this._toggleBallotIntroFollowIssues(this);}}>
         <Modal.Body>
+          <div className="intro-modal__close">
+            <a onClick={this._toggleBallotIntroFollowIssues} className="intro-modal__close-anchor">
+              <img src="/img/global/icons/x-close.png" alt="close" />
+            </a>
+          </div>
           <Slider dotsClass="slick-dots intro-modal__gray-dots" className="calc-height" ref="slider" {...slider_settings}>
-            <div key={1}><BallotIntroFollowIssues next={this._nextSliderPage}/></div>
-            <div key={2}><BallotIntroFollowAdvisers next={this._nextSliderPage}/></div>
-            <div key={3}><BallotIntroPositionBar next={this._nextSliderPage}/></div>
+            <div key={1}><BallotIntroFollowIssues next={this._nextSliderPage} close={this._toggleBallotIntroFollowIssues}/></div>
```

Comment:
Hi Keith, did you mean to remove this prop "close" being passed into BallotIntroFollowIssues, and the other components used by the Slider for the HeaderGettingStartedBar?